frontend/src/components/Terminal/Terminal.jsx

```diff
@@ -43,14 +69,13 @@ export default class Terminal extends React.Component {
     }
 
     rowRenderer = ({ index, key, style }) => {
+        const { searchText, currentSearchRow } = this.props;
+        const searchWords = searchText ? [searchText] : [];
         return (
             <pre key={ key } style={ style }>
                 <Highlighter
-                    highlightClassName="error"
-                    searchWords={ [
-                        "ERROR",
-                        "WARNING",
```

Comment:
This used to highlight the words "error" and "warning" wherever they appeared in the log, which was pretty useless.

Repurpose `<Highlighter>` for search highlighting.